src/main/webapp/js/actions/index.js

```diff
@@ -130,21 +130,25 @@ export const fetchEntity = entityType => (dispatch) => {
   });
 };
 
-export const deleteEntity = (entityType, entityId, callback) => {
+export const deleteEntity = (entityType, entityId, callback, onFail) => (dispatch) => {
```

Comment:
if I understand this correctly when delete fails you are moving user to the list record view (entity table), this should not be the case, he should stay on the edit record view and see the error